srcpkgs/dlib/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'dlib'
+pkgname=dlib
+version=19.18
+revision=1
+build_style=cmake
+configure_args=Release
```

Comment:
that's not doing anything useful. [see here](https://github.com/davisking/dlib/blob/master/dlib/CMakeLists.txt#L48)